src/test/java/com/google/api/generator/engine/ast/JavaDocCommentTest.java

```diff
@@ -29,6 +29,21 @@ public void createJavaDocComment_basic() {
     assertEquals(javaDocComment.comment(), content);
   }
 
+  @Test
+  public void createJavaDocComment_specialCharacter() {
+    JavaDocComment javaDocComment =
+        JavaDocComment.builder()
+            .addComment("Service comment may include special characters: <>&\"`'@")
+            .addParagraph("title: GetBigBook: 'War and Peace'")
+            .setThrows("RunTimeException", "This is an unexpected block end */")
+            .build();
+    String expected =
+        "Service comment may include special characters: <>&\"`'@\n"
+            + "<p> title: GetBigBook: 'War and Peace'\n"
+            + "@throws RunTimeException This is an unexpected block end * /";
+    assertEquals(javaDocComment.comment(), expected);
+  }
+
   @Test
```

Comment:
What about some test cases for comments that would have failed before?

Reply:
Or comments with `/**` or `*/` in the middle?

Reply:
added more special characteres `\t\b\r\n` which would have failed without the escaper.

For `/**` and `*/`, `/**` is not special because it will not break us, `/** this is a test /** comment` is valid. `*/` is tested in throw description, the whole java doc comment will be like:
```
/**
* Service comment may include special characters: <>\t\b\r&\"\n`'@
* <p> title: GetBigBook: 'War and Peace'
* @throws RunTimeException This is an unexpected block end * /
*/
```

Reply:
Note that we _may_ need to replace those with different characters if we perform HTML escaping (see next comment). PTAL at the baseline file to investigate how we handled these characters, if at all.

Reply:
Thanks, I will take investigation of the HTML characters escaping and make a separate PR after this.